docs/tezos_domains.md

```diff
@@ -18,7 +18,7 @@ If you have an address and you want to find the domain tied to it, the storage o
 import { TezosToolkit } from "@taquito/taquito";
 import { bytes2Char } from "@taquito/utils";
 
-const domainContractAddress = "KT1PfBfkfUuvQRN8zuCAyp5MHjNrQqgevS9p";
+const domainContractAddress = "KT1GBZmSxmnKJXGMdMLbugPfLyUPmuLSMwKS";
```

Comment:
As the domainContractAddress is deployed on mainnet, could you replace the RPC URL passed in the TezosToolkit in each example to use mainnet instead of "https://testnet-tezos.giganode.io"?